src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.top = '50%';
+spider.style.left = '50%';
+spider.style.transform = 'translate(-50%, -50%)';
```

Comment:
It's boring solution, let's use `clientWidth` and so on instead

Reply:
I thinks it's the best solution,cuz i changed only css properties